mllib/src/main/scala/org/apache/spark/mllib/util/NumericParser.scala

```diff
@@ -19,6 +19,8 @@ package org.apache.spark.mllib.util
 
 import java.util.StringTokenizer
 
+import org.apache.commons.lang.StringUtils.isBlank
```

Comment:
Don't use commons-lang -- it's not actually a dependency. I wouldn't even use lang3. You don't need a utility class for something this simple